Makefile

```
@@ -10,6 +10,7 @@ endif
 GO                  := GO111MODULE=on go
 GOBUILD             := $(GO) build $(BUILD_FLAG) -tags codes
 GOTEST              := $(GO) test -v --count=1 --parallel=1 -p=1
+TEST_CLEAN			:= rm -rf /tmp/*raft*
```

```suggestion
TEST_CLEAN			:= rm -rf /tmp/*test-raftstore*
```
And please align the indentation